### PR TITLE
Update readme and file generators.

### PR DESCRIPTION
**Pull request by cutandpastey**

### CODE REVIEW

File: bin/index [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-//template data
-var tmplData = {
-  componentName: componentName,
-  capitalizeFirstChar: capitalizeFirstChar
+program
+  .version(pkgInfo.version)
+  .option('-m, --mixin-name [mixin-name]', 'Mixin name')
+  .option('-c, --component-name [component-name]', 'Component name')
+  .option('-d, --directive-name [directive-name]', 'Directive name')
+  .parse(process.argv);
```
Comment (robinedman) [inline bin/index:23]: These don't seem to match what's in the README

**Comment by robinedman** [comment]
Very much like the thorough docs! Good work with that.

**Comment by cutandpastey** [comment]
Yeah, I removed the `directive-name` option. Others should match.

**Comment by robinedman** [comment]
:+1:

**Pull request merged**